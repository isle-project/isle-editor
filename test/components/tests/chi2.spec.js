// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import Chi2Test from '@isle-project/components/tests/chi2';
import heartdisease from '../fixtures/heartdisease.json';


// TESTS //

describe( '<Anova />', function test() {
	it( 'renders test result', () => {
		const output = renderer.create( <Chi2Test
			data={heartdisease}
			var1="Drugs"
			var2="Complications"
		/> );
		expect( output ).toMatchSnapshot();
	});
});
