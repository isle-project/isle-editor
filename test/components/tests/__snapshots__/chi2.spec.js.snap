// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Anova /> renders test result 1`] = `
<div
  style={
    {
      "overflowX": "auto",
      "width": "100%",
    }
  }
>
  <label>
    hypothesis-test-for-independence
    :
  </label>
</div>
`;
